### PR TITLE
Get rid of legacyscheme from oc and move oc to staging

### PR DESCRIPTION
**Pull request by soltysh**
/assign @deads2k 

let's see how this goes...

### CODE REVIEW

**Comment by deads2k** [comment]
still need accept empty group and v1 input from serialized files.

**Comment by soltysh** [comment]
@deads2k cross your fingers, if this is green we're fully in staging :tada:

File: Makefile [hunk 1]
```diff
@@ -33,7 +33,7 @@ JUNIT_REPORT ?= true
 # Example:
 #   make
 #   make all
-#   make all WHAT=cmd/oc GOFLAGS=-v
+#   make all WHAT=cmd/openshift-tests GOFLAGS=-v
```
Comment (mfojtik) [inline Makefile]: @soltysh i would not use openshift-tests as an example :D but 🤷‍♂

Comment (soltysh) [reply Makefile]: well, I'm not sure what's else left here :wink:

**Comment by deads2k** [comment]
/lgtm

Please add a unit test for mixed openshift and kube resources for convert and get it passing.

**Comment by soltysh** [comment]
/hold
integration still has issue with image conversions, I'm looking into it atm

**Pull request merged**